src/ai/backend/manager/api/etcd.py

```diff
@@ -21,13 +24,57 @@
 log = BraceStyleAdapter(logging.getLogger(__spec__.name))  # type: ignore[name-defined]
 
 
+KNOWN_SLOT_METADATA: Mapping[str, AcceleratorMetadata] = {
+    "cpu": {
+        "slot_name": "cpu",
+        "description": "CPU",
+        "human_readable_name": "CPU",
+        "display_unit": "Core",
+        "number_format": "#,###",
+        "display_icon": "cpu",
+    },
+    "mem": {
+        "slot_name": "ram",
+        "description": "Memory",
+        "human_readable_name": "RAM",
+        "display_unit": "GiB",
+        "number_format": "#,###",
+        "display_icon": "cpu",
+    },
+    "cuda": {
+        "slot_name": "cuda.device",
+        "human_readable_name": "GPU",
+        "description": "CUDA-capable GPU",
+        "display_unit": "GPU",
+        "number_format": "#,###",
+        "display_icon": "gpu1",
+    },
+}
+
+
 async def get_resource_slots(request: web.Request) -> web.Response:
     log.info("ETCD.GET_RESOURCE_SLOTS ()")
     root_ctx: RootContext = request.app["_root.context"]
     known_slots = await root_ctx.shared_config.get_resource_slots()
     return web.json_response(known_slots, status=200)
 
 
+async def get_resource_metadata(request: web.Request) -> web.Response:
+    log.info("ETCD.GET_RESOURCE_METADATA ()")
+    root_ctx: RootContext = request.app["_root.context"]
+    available_slot_metadata: Dict[str, Any] = {}
+
+    async with root_ctx.db.begin_readonly() as conn:
+        query = sa.select([agents.c.compute_plugins]).select_from(agents).where()
+        result = await conn.execute(query)
+        for row in result:
+            for key, value in row["compute_plugins"].items():
+                if _data := value.get("metadata", KNOWN_SLOT_METADATA.get(key)):
```

Comment:
What happens if there is no metadata in both the value and `KNOWN_SLOT_METADATA`?
Let's add a little more friendly error description.

Reply:
I've ignored raising error for slot types without metadata intentionally. Added a brief description about these as a block of comment.

Reply:
Um, `dict.get()`'s default second argument is `None`, not `{}`.